tests/installation/registration/rmt_registration_trust_import.pm

```diff
@@ -0,0 +1,20 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register the system via local RMT in the installer with server url
+# and enabling update repositories.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
```

Comment:
you don't need to import the lib when using `$testapi`.

Reply:
Is that true? I copied the imports based on already existing `register_via_scc.pm` - should I create a second commit and remove it from there as well?